src/index.js

```diff
@@ -141,6 +154,11 @@ class TransportEmailPlugin {
     this._unset()
   }
 
+  test ({to}) {
+    return this._sendEmail({to: to,
+      subject: 'Configuration test',
+      markdown: '**Configuration works**'})
```

Comment:
Closing symbols (`}`, `]`, `)`) should be on a line with the same indent level than their matching opening symbol.